### PR TITLE
Bisonquote toy

### PR DESCRIPTION
**Pull request by SoyBison**
I made a message of the day script a while ago that gets quotes. I figured I may as well open the api. Then, out of sheer boredom I added a grace interface. An intergrace if you will.

### CODE REVIEW

File: bot/extensions/fun/fun_cog.py [hunk 1]
```diff
@@ -40,6 +41,30 @@ async def quote_command(self, ctx):
 
         await ctx.send(embed=embed)
 
+    @command(name='bisonquote', help='Sends a quote from SoyBison\'s quote server.')
+    async def bison_quote(self, ctx):
+        response = get(
```
Comment (PenguinBoi12) [inline bot/extensions/fun/fun_cog.py:46]: I don't think the line is long enough. You could just write that in one line

File: setup.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
         'nltk',
         'discord-pretty-help',
         'requests',
+        'psycopg2',
```
Comment (PenguinBoi12) [inline setup.py:24]: We removed that before because it caused issue for some people.

**Review by PenguinBoi12 — CHANGES_REQUESTED** [review]
Fix the 2 comment and you can merge

**Review by SoyBison — COMMENTED** [review]
bap

**Review by PenguinBoi12 — APPROVED** [review]

**Pull request merged**